customTests/azure_common.nhc

```diff
@@ -3,6 +3,12 @@
 export ALL_REDUCE_PATH=/opt/nccl-tests/build/all_reduce_perf
 export HPCX_DIR=/opt/hpcx
 
+function get_sku(){
+   SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text" | tr '[:upper:]' '[:lower:]' | sed 's/standard_//')
```

Comment:
Is this an ARM endpoint? Is there a URL to use rather than an IP?

Reply:
This endpoint is to the Azure Instance Metadata Service: https://learn.microsoft.com/en-us/azure/virtual-machines/instance-metadata-service?tabs=linux

I followed the usage as described in that doc. I am not aware of a URL for this.